nomenclature/definitions/variable/emissions/emissions.yaml

```
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emissions|CO2|Electricity:
```

I suggest to change this for clarification that this is not related to the consumption of electricity (and also implement for the other variables)
```suggestion
Emissions|CO2|Electricity Generation:
```